extensions/resource-deployment/src/interfaces.ts

```diff
@@ -190,6 +190,17 @@ export interface NotebookInfo {
 	linux: string;
 }
 
+export enum OsType {
+	win32 = 1, // Keep this 1 based, so that only falsy values are the ones that are undefined (OsType["unknown"] returns undefined) so easy to map to others.
```

Comment:
I thought we agreed to use the approach that is linux edition agnostic, right?

Reply:
yeah - I can fold the aix, freebsd,openbsd and sunos down to others, as in the implementation I do not do anything about them.

Reply:
removed the extra value of OsType